spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
```

Comment:
I see that you added a `rails_helper.rb` Is not that conflicting with the `spec_helper.rb` wa already have?